Remotion/Web/Core/UI/Controls/WebButton.cs

```diff
@@ -547,6 +571,30 @@ public virtual string CssClassThemed
       get { return CssClassDefinition.Themed; }
     }
 
+    /// <summary>
+    /// Gets the CSS-Class applied to the <c>div</c> when the <see cref="ButtonType"/> is set to <see cref="Controls.ButtonType"/>.<see cref="Controls.ButtonType.Primary"/>.
+    /// </summary>
+    /// <remarks> 
+    ///   <para> Class: <c>primary</c>. </para>
+    ///   <para> Applied in addition to the regular CSS-Class.</para>
+    /// </remarks>
+    public virtual string CssClassPrimary
+    {
+      get { return "primary"; }
```

Comment:
Should we move those to CssClassDefinition as `ButtonTypePrimary` and `ButtonTypeSupplemental`?